### DISCUSSION TITLE
Index generator for importer

### DISCUSSION THREAD

**Discussion by nnhjy** [Spine-project/Spine-Toolbox#1717] [Discussion]
Hi, if we want to import a list of values having no index from a csv/excel file as time-series data, adding indices to the list 
 on importing is quite inconvenient. Please see the attached project. Perhaps we should give the option to create indices in the importer interface.
[BuildTimeseries.zip](https://github.com/Spine-project/Spine-Toolbox/files/8798443/BuildTimeseries.zip)

**Comment by jkiviluo** [comment]
Could be something similar to the way regex is done in the exporter side.
